examples/interfaces/cw4/src/lib.rs

```diff
@@ -1,44 +1,81 @@
 use cosmwasm_std::{Response, StdError};
-
-use sylvia::types::{ExecCtx, QueryCtx};
+use sylvia::types::{CustomMsg, CustomQuery, ExecCtx, QueryCtx};
 use sylvia::{interface, schemars};
 
 #[interface]
-#[sv::custom(msg=cosmwasm_std::Empty, query=cosmwasm_std::Empty)]
 pub trait Cw4 {
     type Error: From<StdError>;
+    type ExecC: CustomMsg;
+    type QueryC: CustomQuery;
+    type MemberCustomMsg: CustomMsg;
+    type ListMembersCustomMsg: CustomMsg;
+    type TotalWeightCustomMsg: CustomMsg;
+    type AdminCustomMsg: CustomMsg;
+    type HooksCustomMsg: CustomMsg;
```

Comment:
Remove.

The queries most likely should not return the `Response` and instead some custom response type

Please add those messages
https://github.com/CosmWasm/cw-plus/blob/main/packages/cw4/src/query.rs